dockertest/output.py

```diff
@@ -767,25 +791,37 @@ def mustfail(cmdresult, failmsg=None):
     :type cmdresult: object convertible to string with variable exit_status
     :param failmsg: Additional messages for describing problem when cmd fails.
```

Comment:
Ahh, we need a docstring update here for ``expected_status``.  Might be good to mention there that it will be used in place of  ``failmsg`` if it's a string instead [of something else].